deps/rabbit/src/rabbit_deprecated_features.erl

```diff
@@ -118,6 +118,7 @@
 -export([extend_properties/2,
          should_be_permitted/2,
          enable_underlying_feature_flag_cb/1]).
+-export([list/1]).
```

Comment:
`list/1` would be part of the public API of this module, so it can be listed in the existing `-export()` directive.